index.js

```diff
@@ -151,6 +151,19 @@ MetaInspector.prototype.getMetaDescription = function()
 	return this;
 }
 
+MetaInspector.prototype.elemContainsTag = function(elem,tagName) {
+	if (elem.tagName && elem.tagName === tagName ) {
+		return true;
+	} else if ( elem.children != undefined ) {
+		for ( let i = 0; i < elem.children.length; i++) {
```

Comment:
Looks like this is causing the Travis CI build to fail since `let` is only supported in Node 6 and this library targets as far back as Node 0.10. Can you replace the `let` declaration with `var`?